### DISCUSSION TITLE
Export only last name

### DISCUSSION THREAD

**Discussion by divyaranjan1905** [emacs-citar/citar#843] [Q&A]
How to export only the last name of the author and year? Currently after inserting with citar I have full names of authors within the text, I do not want that. It should be either the last name of the author and two last names of two authors, more than two it should be first author et. al.

I changed `citar-org-styles-format` to `short` but that didn't change anything.

**Comment by bdarcus** [comment]
WDYM by "export"? And in what major mode?

Depending on your answers, the solution may not be within citar. The only place citar handles formatting of names is in the minibuffer display, and in the default for `citar-copy-reference`.

**Reply by divyaranjan1905** [reply]
I'm doing `org-export` in an Org file. With `citar` as the processor.

**Reply by bdarcus** [reply]
Citar is the insert processor, then; not export. 

Perhaps you're using the `oc-csl` export processor there? If yes, you need to find and specify a CSL file that gets the formatting you want.

Regardless, this is an org-cite issue; not citar.

**Reply by divyaranjan1905** [reply]
Ah. Got it. Thanks.
